brave/brave-core
# Respect DOH Dns config in IPFS

Resolves https://github.com/brave/brave-browser/issues/22409
DOH url is taken from network service request. After DNS config updates IPFS daemon restarts.
Sec review: https://github.com/brave/security/issues/949

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1) Check ipfs opens pages.
2) Enable DOH.
3) Check ipfs opens pages and check that there are no default dns requests to ipfs nodes.
4) Write custom DNS server and check that ipfs still works.
